### ISSUE TITLE
https://github.com/open-mmlab/mmdetection/blob/main/mmdet/models/losses/eqlv2_loss.py 中的代码语法问题

### ISSUE THREAD

**Issue by xiaoche-24** [open-mmlab/mmdetection#11426]
**Describe the bug**
在eqlv2_loss.py中，121行代码感觉是有语法问题：
`def forward(self,
                cls_score: Tensor,
                label: Tensor,
                weight: Optional[Tensor] = None,
                avg_factor: Optional[int] = None,
                reduction_override: Optional[Tensor] = None) -> Tensor:
        """`Equalization Loss v2 <https://arxiv.org/abs/2012.08548>`_

        Args:
            cls_score (Tensor): The prediction with shape (N, C), C is the
                number of classes.
            label (Tensor): The ground truth label of the predicted target with
                shape (N, C), C is the number of classes.
            weight (Tensor, optional): The weight of loss for each prediction.
                Defaults to None.
            avg_factor (int, optional): Average factor that is used to average
                the loss. Defaults to None.
            reduction_override (str, optional): The reduction method used to
                override the original reduction method of the loss.
                Options are "none", "mean" and "sum".

        Returns:
           Tensor: The calculated loss
        """
        self.n_i, self.n_c = cls_score.size()
        self.gt_classes = label
        self.pred_class_logits = cls_score

        def expand_label(pred, gt_classes):
            target = pred.new_zeros(self.n_i, self.n_c)
===》 target[torch.arange(self.n_i), gt_classes] = 1  《===
            return target

        target = expand_label(cls_score, label)
`
如上   =》 target[torch.arange(self.n_i), gt_classes] = 1 《=   所指的那行代码，函数expand_label输入为forward的输入cls_score、label，两者均为（N,C）的tensor，在expand_label中，首先定义target是尺寸和cls_score一样的全0矩阵，但是在下面一行target[torch.arange(self.n_i), gt_classes] = 1中，为什么索引gt_classes，这难道不是一个语法错误？
